fastai/tabular/model.py

```diff
@@ -20,16 +22,30 @@ def _one_emb_sz(classes, n, sz_dict=None):
     return n_cat,sz
 
 # Cell
-def get_emb_sz(to, sz_dict=None):
-    "Get default embedding size from `TabularPreprocessor` `proc` or the ones in `sz_dict`"
+def get_emb_sz(
+    to:(Tabular, TabularPandas), # Tabular or TabularPandas object
```

Comment:
```suggestion
    to:(Tabular, TabularPandas),
```
No need to repeat ourselves. The docment doesn't provide value so it's not needed